beginner_source/onnx/onnx_registry_tutorial.py

```diff
@@ -231,33 +230,24 @@ def custom_aten_gelu(input_x, approximate: str = "none"):
 
 
 ######################################################################
-# Let's inspect the model and verify the model uses :func:`custom_aten_gelu` instead of
-# :class:`aten::gelu`. Note the graph has one graph nodes for
-# ``custom_aten_gelu``, and inside ``custom_aten_gelu``, there is a function
-# node for ``Gelu`` with namespace ``com.microsoft``.
+# Let's inspect the model and verify the model uses op_type ``Gelu``
+# from namespace ``com.microsoft``.
+#
+# Note that :func:`custom_aten_gelu` does not exist in the graph, because
+# the funtions with less than 3 operators are inlined automatically.
```

Comment:
```suggestion
# .. note::
#     :func:`custom_aten_gelu` does not exist in the graph because
#     functions with fewer than three operators are inlined automatically.
```